DQMOffline/L1Trigger/src/L1TMuonDQMOffline.cc

```diff
@@ -90,10 +90,10 @@ L1TMuonDQMOffline::EtaRegion MuonGmtPair::etaRegion() const {
 }
 
 double MuonGmtPair::getDeltaVar(const L1TMuonDQMOffline::ResType type) const {
-    if (type == L1TMuonDQMOffline::kResPt)      return gmtPt() - pt();
-    if (type == L1TMuonDQMOffline::kRes1OverPt) return 1/gmtPt() - 1/pt();
-    if (type == L1TMuonDQMOffline::kResQOverPt) return gmtCharge()/gmtPt() - charge()/pt();
-    if (type == L1TMuonDQMOffline::kResPhi)     return gmtPhi() - phi();
+    if (type == L1TMuonDQMOffline::kResPt)      return (gmtPt() - pt()) / pt();
+    if (type == L1TMuonDQMOffline::kRes1OverPt) return (pt() - gmtPt()) / gmtPt(); // (1/gmtPt - 1/pt) / (1/pt)
```

Comment:
why this change? The name Res1OverPt does not seem any more true...

Reply:
@thomreis I see what you mean in the comment, it does not seem completely straightforward but ok